packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics-latencies.ts

```diff
@@ -184,6 +194,30 @@ describe("internal-plugin-metrics", () => {
       assert.deepEqual(cdl.getTotalJMT(), 45);
     });
 
+    it('calculates getTotalMediaJMT correctly', () => {
+      cdl.saveTimestamp('internal.client.meeting.click.joinbutton', 5);
+      cdl.saveTimestamp('internal.client.meeting.interstitial-window.showed', 8);
+      cdl.saveTimestamp('internal.client.interstitial-window.click.joinbutton', 10);
+      cdl.saveTimestamp('client.locus.join.request', 12);
+      cdl.saveTimestamp('client.locus.join.response', 20);
+      cdl.saveTimestamp('internal.host.meeting.participant.admitted', 24)
+      cdl.saveTimestamp('client.ice.start', 30);
+      cdl.saveTimestamp('client.ice.end', 40);
+      assert.deepEqual(cdl.getTotalMediaJMT(), 27);
+    })
+
+    it('calculates getTotalMediaJMT correctly with allowMediaInLobby true', () => {
```

Comment:
it's probably worth keeping the call to saveTimestamp with 'internal.host.meeting.participant.admitted' in this test too, because that's what will happen in reality and to really test that the time when participant is admitted doesn't affect the TotalMediaJMT

Reply:
that's a good point, added